akka-actor-typed-tests/src/test/scala/akka/actor/typed/BehaviorSpec.scala

```diff
@@ -70,7 +71,9 @@ object BehaviorSpec {
   trait Common extends WordSpecLike with Matchers with TypeCheckedTripleEquals {
     type Aux >: Null <: AnyRef
     def behavior(monitor: ActorRef[Event]): (Behavior[Command], Aux)
+    @silent
```

Comment:
maybe use @unused instead?

Reply:
@raboof I merged before you had a chance to do @chbatey's suggestions, didn't notice them due to his LGTM until after. Noted.

Reply:
Would have to be
```scala
def checkAux(@unused signal: Signal, @unused aux: Aux): Unit = ()
```
In which case `@silent("not used"` seems cleaner https://github.com/akka/akka/pull/27433

Reply:
(the reason I'm reluctant to add `@unused` in this case is because the intent is that the parameter *is* actually used in subclasses. Promising a variable is 'unused' in a superclass and then using it anyway in a subclass doesn't feel right)